.github/workflows/style.yml

```diff
@@ -0,0 +1,27 @@
+name: Python style tests
+
+on: [push]
+
+jobs:
+
+  tox:
+    name: tox
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install postgresql-devel
```

Comment:
do we need this?
I don't think we are using any postgresql package here, maybe in e2e we need it.